site/diagrams/sequenceDiagrams/command_control_concept_mqtt.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant MQTT Adapter
+participant AMQP 1.0 Network
+participant Application
+
+Device->MQTT Adapter: 1. send event
+note over Device
+content-type=application/vnd.eclipse-hono-empty-notification
+-> MQTT Adapter will convert this in tdd=-1 (infinitely)
+end note over
+activate MQTT Adapter
+MQTT Adapter->Application: 2. connected notification (ttd=-1)
+activate Application
+Application->MQTT Adapter: 3. transfer("control/TENANT/4711","light","ON")
+MQTT Adapter->Device: 4. "control/light/${reqId}":"ON"
+Device->MQTT Adapter: 5. "control/res/200/${reqId}":"lumen=100"
```

Comment:
I believe the request ID should be *before* the status code in the topic name

Reply:
For what reason? The status is a part of the response from the device, the reqId is just needed for transport.

Reply:
FMPOV this better reflects the *natural semantic ordering* of the information:
> this is the response to request X and it has outcome 200

as opposed to:
> this is a response to a request which has a status of 200, oh and by the way, the request that this is the response to is ${reqId} ...